src/main/kotlin/racing/Car.kt

```diff
@@ -1,24 +1,21 @@
 package racing
 
-import kotlin.random.Random
+private const val DEFAULT_POSITION = 1
+private const val MINIMUM = 4
+private const val MOVING_DISTANCE = 1
 
-data class Car(private var _position: Int = 1) {
-    val position get() = _position
+data class Car(val name: String, private var _position: Int = DEFAULT_POSITION) {
+    val position: Int get() = _position
```

Comment:
```suggestion
    val position: Int 
        get() = _position
```
저는 의도적으로 get()을 내리는 편입니다 :)

Reply:
수정했습니다. https://github.com/inhyeokk/kotlin-racingcar/commit/f8701016ef973d40f91fbbbf1fab7d13d4098f80